### PR TITLE
Update: Add displayName getter in base.ts, and tests for it in column…

### PR DESCRIPTION
**Pull request by holderb5**
Resolves #991 
Resolves the above for local testing, not for travis-ci

## Description
Add in a displayName getter into base.ts fragment, as well as add tests for it and the column display name getter

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/core/addon/models/bard-request-v2/fragments/base.ts [hunk 1]
```diff
@@ -13,6 +13,8 @@ import { canonicalizeMetric } from 'navi-data/utils/metric';
 import NaviMetadataService from 'navi-data/services/navi-metadata';
 import { Parameters, ColumnType } from 'navi-data/adapters/facts/interface';
 import MetadataModelRegistry from 'navi-data/models/metadata/registry';
+// @ts-ignore
```
Comment (kevinhinterlong) [inline packages/core/addon/models/bard-request-v2/fragments/base.ts:16]: I think this should be typed
```suggestion
```

File: packages/core/tests/unit/models/bard-request-v2/fragments/column-test.js [hunk 2]
```diff
@@ -120,4 +120,18 @@ module('Unit | Model | Fragment | BardRequest - Column', function(hooks) {
...
     );
   });
+
+  test('Display Name', async function(assert) {
+    const column = mockModel.columns.objectAt(0);
+
+    column.set('field', 'revenue');
+    column.set('parameters', { currency: 'USD' });
+    column.set('type', 'metric');
+    column.set('alias', 'revenueUSD');
```
Comment (jkusa) [inline packages/core/tests/unit/models/bard-request-v2/fragments/column-test.js:130]: Can we add a non parameter test?

File: packages/core/tests/unit/models/bard-request-v2/fragments/column-test.js [hunk 3]
```diff
@@ -120,4 +120,18 @@ module('Unit | Model | Fragment | BardRequest - Column', function(hooks) {
...
+
+    assert.equal(column.displayName, 'revenueUSD (USD)');
```
Comment (jkusa) [inline packages/core/tests/unit/models/bard-request-v2/fragments/column-test.js:132]: Can we add assert messages?

File: packages/core/addon/models/bard-request-v2/fragments/column.ts [hunk 4]
```diff
@@ -18,7 +18,6 @@ export default class ColumnFragment extends BaseFragment implements Column {
 
   @service naviFormatter!: NaviFormatterService;
 
-  @computed('columnMetadata', 'alias')
```
Comment (kevinhinterlong) [inline packages/core/addon/models/bard-request-v2/fragments/column.ts:21]: The import wasn't removed

File: packages/data/addon/services/navi-formatter.ts [hunk 5]
```diff
@@ -13,7 +13,7 @@ export default class NaviFormatterService extends Service {
     const paramValues = Object.values(allParams);
 
     const name = alias || columnMetadata?.name || '--';
```
Comment (kevinhinterlong) [inline packages/data/addon/services/navi-formatter.ts:15]:
```suggestion
    const name = columnMetadata?.name || '--';
```

File: packages/data/addon/services/navi-formatter.ts [hunk 6]
```diff
@@ -13,7 +13,7 @@ export default class NaviFormatterService extends Service {
...
-    if (paramValues.length) {
+    if (paramValues.length && alias === null) {
       return `${name} (${paramValues.join(',')})`;
     } else {
       return name;
```
Comment (kevinhinterlong) [inline packages/data/addon/services/navi-formatter.ts:19]:
```suggestion
    if (paramValues.length && !alias) {
      return `${name} (${paramValues.join(',')})`;
    } else {
      return alias;
```

Comment (holderb5) [reply packages/data/addon/services/navi-formatter.ts]: So I tried this specific format and it caused a few errors with the tests, so I stuck with the original but changes alias === null to !alias

**Review by jkusa — APPROVED** [review]

File: packages/data/addon/services/navi-formatter.ts [hunk 7]
```diff
@@ -13,7 +13,7 @@ export default class NaviFormatterService extends Service {
     const paramValues = Object.values(allParams);
 
     const name = alias || columnMetadata?.name || '--';
-    if (paramValues.length) {
+    if (paramValues.length && !alias) {
       return `${name} (${paramValues.join(',')})`;
     } else {
       return name;
```
Comment (kevinhinterlong) [inline packages/data/addon/services/navi-formatter.ts:19]: Can we just add something like this at the top of the function. I feel like that would be more explicit than this check
```js
if (alias) {
  return alias;
}
```

Comment (holderb5) [reply packages/data/addon/services/navi-formatter.ts:19]: Sure I'll try that!

**Review by kevinhinterlong — APPROVED** [review]

**Review by jkusa — APPROVED** [review]

**Pull request merged**